src/kili/use_cases/base.py

```diff
@@ -0,0 +1,12 @@
+"""Base class for all use cases."""
+
+from kili.adapters.kili_api_gateway import KiliAPIGateway
+
+
+class BaseUseCases:
+    """Base class for all use cases."""
+
+    kili_api_gateway: KiliAPIGateway
```

Comment:
no need to declare this class attribute

the class can be made abstract maybe, since it should not be instantiated directly

also, existing use cases (project, tags, etc.) should inherit from this class